c-org-abi/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fairmint/c-org-abi",
-  "version": "2.4.17",
+  "version": "2.4.18",
```

Comment:
Actually we should change to 2.4.22.
CI deploys via the release tags, but ATM those are shared with c-org which is at a higher version.

Reply:
This is done.